client/src/dashboard-view.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+// Helpers
+import classNames from 'classnames/bind'
+import apiCall from './api-call'
+import { subscribeUsers, unsubscribeUsers } from './globals'
+// Styles
+import styles from './scss/views/dashboard-view.scss'
+
+const css = classNames.bind(styles)
+
+class Dashboard extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.refreshInstances = this.refreshInstances.bind(this)
+
+    this.state = {
+      instances: [],
+    }
+  }
+
+  componentDidMount() {
+    this.userSub = subscribeUsers(this.refreshInstances)
+  }
+
+  componentWillUnmount() {
+    unsubscribeUsers(this.userSub)
+  }
+
+  async refreshInstances() {
+    const instances = await apiCall('GET', '/instances')
+    this.setState({ instances })
+  }
+
+  render() {
+    const { instances } = this.state
+    const noInstances = () => {
+      if (instances.length < 1) {
+        return (
+          <p>
+            No instances created.
+            Go ahead and create one by clicking on the &quot;New instance&quot; button.
+          </p>
+        )
+      }
+    }
```

Comment:
this returns undefined in in case of `instance.length >= 1` and doesn't need to be a function in the first place. just an inline piece of
```js
{ instances.length < 1 ? (
  <p>
    No instances created.
    Go ahead and create one by clicking on the "New instance" button.
  </p>
) : null }
```
should suffice

Reply:
Agree. Straight line code >>> function calls unless there is a real reason to have a function

Reply:
done